Source/WarpX.H

```diff
@@ -560,6 +560,8 @@ private:
     void LoadBalance ();
 
     void BuildBufferMasks ();
+    void BuildBufferMasksInBox ( const amrex::Box tbx, amrex::Array4<int> &msk,
```

Comment:
Could you add a Doxygen-readable docstring to this function, either here or in the cpp file?